packages/edit-site/src/components/left-sidebar/navigation-panel/index.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	__experimentalNavigation as Navigation,
+	__experimentalNavigationGroup as NavigationGroup,
+	__experimentalNavigationItem as NavigationItem,
+	__experimentalNavigationMenu as NavigationMenu,
+} from '@wordpress/components';
+import { getBlockType, getBlockFromExample } from '@wordpress/blocks';
+import { BlockPreview } from '@wordpress/block-editor';
+
+const NavigationPanel = () => {
+	const [ activeItem, setActiveItem ] = useState( 'item-1' );
+	const [ activeMenu, setActiveMenu ] = useState( 'root' );
+	const [ showPreview, setShowPreview ] = useState( false );
+
+	return (
+		<>
+			<div style={ { position: 'relative', height: '100%' } }>
+				<Navigation
+					activeItem={ activeItem }
+					activeMenu={ activeMenu }
+					onActivateMenu={ setActiveMenu }
+				>
+					<NavigationMenu title="Home">
+						<NavigationItem
+							item="item-back"
+							title="Back to dashboard"
+							href="index.php"
+						/>
+
+						<NavigationGroup title="Group 1">
+							<NavigationItem
+								item="item-preview"
+								title="Hover to show preview"
+								onMouseEnter={ () => setShowPreview( true ) }
+								onMouseLeave={ () => setShowPreview( false ) }
+							/>
```

Comment:
How do you feel about removing all demo content except for "Back to dashboard" and "Hover to show preview"?
Everything else is just bloating the PR, and we can just look at the Storybook for usage examples.

Reply:
Removed them but I left a `NavigationGroup` just in case. Once this merged I'll put some real data in the navigation sidebar so this phase is kinda temporarily.